src/pudl_zenodo_storage/frictionless/epacamd_eia.py

```diff
@@ -17,7 +17,7 @@ def datapackager(dfiles):
         https://frictionlessdata.io/specs/data-package/
     """
     return DataPackage.from_resource_archiver(
-        DataSource.from_id("epacems_unitid_eia_plant_crosswalk"),
+        DataSource.from_id("epacamd_eia"),
```

Comment:
For this to work you'll need to define the new data source in the PUDL repo, if you haven't already. This repository relies on the `dev` branch of PUDL for its metadata so we need to get those changes merged in there first.

It looks like you'll need to rename the old `epacems_unitid_eia_plant_crosswalk` source that's in `pudl/metadata/sources.py` to the new `epacamd_eia` and update any metadata defined there as appropriate.

Reply:
Hmmm, I already did this and then ran `pip install -e ./` from the pudl repo in the `pudl-zenodo-storage` environment

Reply:
Okay this is updated on `dev` and it seems to be working.